### PR TITLE
Segment now accepts multiple graphs (onTop has to be true tho), changed legend to be on the right and are circles, after scrolling down a bit, button pops out in the bottom right, section title component

### PR DESCRIPTION
**Pull request by eevanwong**
Prob should have just put the content in here but I am just not thinking atm.

### CODE REVIEW

**Comment by shivam-sh** [comment]
Can you attach screenshots showing your changes?

File: components/layout/segment/segment.js [hunk 1]
```diff
@@ -26,7 +32,18 @@ export default function Segment(props) {
...
+                {/* if doubleGraph is true (or essentially there is more than one graph, then it will conditionally render with a new div) */}
+                {doubleGraph ? 
+                props.graph.props.children.map((graph,i) => {
+                    return(
+                    <div className={styles.graphContainer} key={i}> 
+                        {graph}
+                    </div> 
+                )})
+                
+                : props.graph
```
Comment (shivam-sh) [inline components/layout/segment/segment.js:44]: Can we swap out the ternary operator for an if statement? I think that would be better for readability

File: components/charts/doughnut.js [hunk 2]
```diff
@@ -56,12 +57,19 @@ export default function DoughnutChart(props) {
           options={{
             responsive: true,
             maintainAspectRatio: true,
-            legend: { display: false },
+            legend: { 
+              position:"right",
+              labels: {
+                usePointStyle: true,
```
Comment (shivam-sh) [inline components/charts/doughnut.js:63]: Can the legend be made of rounded squares? I think that could fit in really well

Comment (eevanwong) [reply components/charts/doughnut.js:63]: aight sure ill look into that

File: components/layout/segment/segment.js (hunk 1 above)
Comment (eevanwong) [reply components/layout/segment/segment.js:44]: yeah sure, it would be good for people to know that tho

**Comment by eevanwong** [comment]
vid of changes

https://user-images.githubusercontent.com/71536798/115479498-83413100-a216-11eb-9a44-ee12b9576cf4.mp4

File: components/layout/segment/segment.js (hunk 1 above)
Comment (shivam-sh) [reply components/layout/segment/segment.js:44]: It’s great to know but the syntax gets a bit messy when split across multiple lines

**Comment by shivam-sh** [comment]
Looks good, I think it should be ready to merge after addressing the comments and asking the design team about the scroll button

File: pages/profile/demographics/index.js [hunk 3]
```diff
@@ -15,14 +17,41 @@ export default function Demographics() {
...
-        <Bar datatype="Religion" />
-        <DoughnutChart datatype="Year Born" />
-        <Bar datatype="Sexuality" />
+        </>
+        } onTop={true}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum 
+          
+        </Segment>
+
+        <Segment title="Religion" graph={<DoughnutChart datatype="Religion" />}>
```
Comment (parminpatel28) [inline pages/profile/demographics/index.js:47]: you might want to delete this or rebase with develop since we are not using doughnut for religion

Comment (eevanwong) [reply pages/profile/demographics/index.js:47]: yeah aight got it.

Comment (eevanwong) [reply pages/profile/demographics/index.js:47]: Im gonna delete this because I rebased it but it didnt change, r u sure the change of graph is on develop?

Comment (parminpatel28) [reply pages/profile/demographics/index.js]: ya I delete that part on develop but its staying for you is because you have made changes to it as well.

**Comment by eevanwong** [comment]
Yo I remember shivam wanted rounded squares, thing with that is that it requires having a custom legend, which means I will also need to program the functionality of clicking on the  legend. Also, hovering over the parts are also a nightmare because its lagging out for some reason.

Idk, I researched it and it seems a whole lot of work for simply changing a circle to a rounded square

**Comment by parminpatel28** [comment]
> Yo I remember shivam wanted rounded squares, thing with that is that it requires having a custom legend, which means I will also need to program the functionality of clicking on the legend. Also, hovering over the parts are also a nightmare because its lagging out for some reason.
> 
> Idk, I researched it and it seems a whole lot of work for simply changing a circle to a rounded square

round squares for what?

**Comment by eevanwong** [comment]
> 
> 
> > Yo I remember shivam wanted rounded squares, thing with that is that it requires having a custom legend, which means I will also need to program the functionality of clicking on the legend. Also, hovering over the parts are also a nightmare because its lagging out for some reason.
> > Idk, I researched it and it seems a whole lot of work for simply changing a circle to a rounded square
> 
> round squares for what?

Rounded squares for the legend instead of the circles

**Review by parminpatel28 — APPROVED** [review]

**Pull request merged**